### PR TITLE
Add Getting_Started_with_Kotlin.yml file

### PR DESCRIPTION
**Pull request by PrajjawalBanati**

### CODE REVIEW

**Comment by PrajjawalBanati** [comment]
@ritwik12 Thank you for suggesting the changes 👍

File: _data/projects/Kotlin_Language.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: Kotlin_language
+desc: Kotlin Started from basics 
+site: https://github.com/PrajjawalBanati/Kotlin_Language
+tags:
+- kotlin
+- kotlin language
+- kotlin multiplatform
+- kotlin OOPs
+- kotlin FP
```
Comment (ritwik12) [inline _data/projects/Kotlin_Language.yml]: @PrajjawalBanati Please correct these:

```
1 errors found processing projects:
 - _data/projects/Kotlin_Language.yml:
    - Tag verification failed - Tag 'kotlin language' contains spaces or '_' (should use '-' instead)
Tag 'kotlin multiplatform' contains spaces or '_' (should use '-' instead)
Tag 'kotlin OOPs' contains uppercase characters
Tag 'kotlin OOPs' contains spaces or '_' (should use '-' instead)
Tag 'kotlin FP' contains uppercase characters
Tag 'kotlin FP' contains spaces or '_' (should use '-' instead)
##[error]Process completed with exit code 255.
```

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by ritwik12** [comment]
@PrajjawalBanati Also, can you please change the name of the project? I don't want people to get confused with https://kotlinlang.org/.

After changing name in YML, please change the title of this PR also from `Added the .yml file` to
'Add name.yml' whatever name you choose

**Comment by PrajjawalBanati** [comment]
Thank you so much @ritwik12 💯

File: _data/projects/Getting_Started_with_Kotlin.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+desc: Kotlin Started from basics 
+site: https://github.com/PrajjawalBanati/Kotlin_Language
+tags:
+- kotlin
+- kotlin-language
+- kotlin-multiplatform
+- kotlin-android
+- kotlin-functional-programming
+upforgrabs:
+  name: up-for-grabs
```
Comment (ritwik12) [inline _data/projects/Getting_Started_with_Kotlin.yml]: @PrajjawalBanati should be help wanted as per the link

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/Getting_Started_with_Kotlin.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+desc: Kotlin Started from basics 
+site: https://github.com/PrajjawalBanati/Kotlin_Language
+tags:
+- kotlin
+- kotlin-language
+- kotlin-multiplatform
+- kotlin-android
+- kotlin-functional-programming
+upforgrabs:
+  name: help-wanted
```
Comment (ritwik12) [inline _data/projects/Getting_Started_with_Kotlin.yml]: @PrajjawalBanati  `help wanted`, not help-wanted

Comment (ritwik12) [reply _data/projects/Getting_Started_with_Kotlin.yml]: See it is https://github.com/PrajjawalBanati/Kotlin_Language/labels/help%20wanted

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by ritwik12** [comment]
@PrajjawalBanati Update PR title please

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@PrajjawalBanati Thanks!!

**Pull request merged**